Demo/src/main/java/com/paypal/android/di/ApiModule.kt

```diff
@@ -0,0 +1,18 @@
+package com.paypal.android.di
```

Comment:
Potentially dumb question - what does di mean here?

Reply:
Not dumb at all, is short for Dependency Injection, would you like to see it in a different package or with a more explicit name?

Reply:
Hmmm.... open to other opinions here (maybe di is obvious) but I think dependency-injection might be more helpful as a package name

Reply:
My vote is for keeping the package name `di`. I've seen this pattern the most in projects with dependency injection.